pages/_document.tsx

```diff
@@ -7,7 +7,7 @@ class MyDocument extends Document {
       <Html>
         <Head>
           <link
-            href="https://eds-static.equinor.com/font/equinor-font.css"
+            href="https://cdn.eds.equinor.com/font/equinor-font.css"
```

Comment:
Hva brukes _document til?

Reply:
`_document.tsx`is used to override the default HTML structure of a Next.js application.